### PR TITLE
Minor tweak to create nf-core tutorial

### PR DESCRIPTION
**Pull request by jfy133**
Phil went a bit too fast, they were not able to change the root folder in the gitpod VSCode side bar/explorer.

This is because the key command he referenced wasn't correct.

Not sure why but for Linux (me locally), and people on Browser VSCode (Mac OS) have <kbd>Ctrl</kbd> + <kbd>O</kbd> instead

image

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1288"></a>

### CODE REVIEW

**Comment by jfy133** [comment]
@nf-core-bot fix linting

**Comment by ewels** [comment]
@jfy133 could you put that screenshot in please? See the other images for file paths.

**Comment by jfy133** [comment]
> @jfy133 could you put that screenshot in please? See the other images for file paths.

done!

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: markdown/developers/tutorials/creating_with_nf_core.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ To work with a clean directory, you can do the following:
   cd training
   ```
 
-- In the menu top left, select File > Open Folder (<kbd>^</kbd> <kbd>⇧</kbd> <kbd>O</kbd>)
+- In the menu top left, select File > Open Folder (<kbd>^</kbd> <kbd>⇧</kbd> <kbd>O</kbd> or <kbd>^</kbd> <kbd>O</kbd>)
+
+  ![gitpod open folder menu](/assets/markdown_assets/developers/creating_with_nf_core/nfcore_help.svg)
```
Comment (ewels) [inline markdown/developers/tutorials/creating_with_nf_core.md:42]:
```suggestion
  ![gitpod open folder menu](/assets/markdown_assets/developers/creating_with_nf_core/gitpod_open_folder.png)
```

**Review by ewels — APPROVED** [review]
Thanks!

**Pull request merged**